### ISSUE TITLE
Asynchronous tests run twice in standalone

### ISSUE THREAD

**Issue by schickling** [jasmine/jasmine#613]
When running the standalone version of jasmine tests (asynchronous) run twice. The log looks something like this:

```
TestA starts
TestA done asynchronously
TestB starts
TestB done asynchronously
TestA starts
TestA done asynchronously
TestB starts
TestB done asynchronously
TestC starts
TestC done asynchronously
```

As far as I can see the tests/jasmine are not loaded twice.

It might be important to note, that I'm running the jasmine standalone version inside a Cordova application on an iOS device. I initialize the tests like the following, since I need to wait for cordova to finish loading:

``` js
var loadScript = function(src) {
  var el = document.createElement('script')
  el.setAttribute('src', src);
  el.async = false;
  document.body.appendChild(el);
};

var loadTests = function() {
  [
    'lib/jasmine-2.0.0/boot.js',
    'spec/myTest.js',
  ].forEach(loadScript);

  // timeout and window.onload needed to start tests
  setTimeout(function() {
    window.onload();
  }, 1000);
};

document.addEventListener('deviceready', loadTests, false);
```

**Comment by teo-sk** [comment]
@shashankmehta oh my glob you just saved me after 3 hours of debugging :'D

**Comment by OscarGodson** [comment]
Happens for me right now. I have one test:

```
  describe('#scrollToRow', function () {
    var bgt;
    beforeEach(function (done) {
      console.log('==============BEFORE==============')
      bgt = new BGT(defaultOpts);
      bgt.addListener('rowsbuilt', function () { done(); });
    });
    afterEach(function () {
      console.log('==============AFTER==============')
      bgt.destroy();
    });
    it('scrolls to the row so the scrollTop is equal to the row height x index in the table', function (done) {
      // Note for testing we just assume all rows are 10px (see Clusterize stub)
      var els = bgt.getElements();
      console.log('==============TEST==============')
      expect(els.scrollArea.scrollTop).toEqual(0);

      bgt.scrollToRow(2);
      expect(els.scrollArea.scrollTop).toEqual(20);

      bgt.updateRowDisplay(bgt.data.rows.reverse());
      bgt.addListener('rowsbuilt', function () {
        bgt.scrollToRow(2);
        expect(els.scrollArea.scrollTop).toEqual(20);
        done();
      });
    });
  });
```

The output of that is 

![karma debug runner 2016-02-11 15-15-15](https://cloud.githubusercontent.com/assets/20615/12993995/61b3b6cc-d0d2-11e5-8b78-0582634084b9.png)

This makes it _super_ hard to deal with because at least if it duplicated the before/afters I could clean up and the test should work, but these tests are failing because the scrollTops are altered from the previous tests.

**Comment by OscarGodson** [comment]
Oh, and I upgraded Jasmine and Karma like suggested, reset the server, etc and still that happens

**Comment by slackersoft** [comment]
@OscarGodson what version of Jasmine are you using? From what I can see in your test it looks like the `rowsbuilt` listener from the `beforeEach` is being called again during the spec which is causing the spec to execute again. This should have been fixed in #523 in version 2.0.1.

You could also try removing that handler after it is executed the first time and see if that fixes the issue.

Thanks for using Jasmine!

**Comment by p0ma** [comment]
If you are facing this problem in WebStorm: check if you closed your previous debug sessions (browser tabs) :)
